docs/config.md

```diff
@@ -8,7 +8,7 @@ These are the config settings available to jj/Jujutsu.
 The config settings are loaded from the following locations. Less common ways to
 specify `jj` config settings are discussed in a later section.
 
-* `~/.jjconfig.toml` (global)
+* `~/.config/jj/config.toml` (global)
```

Comment:
You'd also want to reword the first paragraph of the " Alternative ways to specify configuration settings" section, around line 454.

Reply:
Done.